docs/zh/api/lynx-api/performance-api/performance-entry/pipeline-entry.mdx

```diff
@@ -171,13 +171,13 @@ paintEnd: number;
 
 基于 UI 和 UITree，完成最终像素化的结束时间戳。该时间戳是一个表示为浮点型的 Unix 时间戳（单位：毫秒），精确到小数点后三位。例如：1739594612307.429。
```

Comment:
What is "UI" and "UI tree"? Find the proper definition in the Lynx spec and reference to them instead. (If the spec is broken, fix the spec)